facts/inventory.py

```diff
@@ -350,14 +351,14 @@ def serveralias(name):
     """generate aliases for servers. Rendered as CNAMES"""
     payload = []
     match name.lower():
-        case "coreexpo":
+        case "core-slave":
```

Comment:
did we get this the other way around? I thought we were going to specify the core by location: `expo` or `conf`? @kylerisse